bin/ncp/BACKUPS/nc-restic-s3-backup.sh

```diff
@@ -0,0 +1,170 @@
+#!/bin/bash
+
+# Back up Nextcloud data to S3-compatible storage via restic
+#
+# Copyleft 2021 by Thomas Heller
+# Copyleft 2017 by Ignacio Nunez Hernanz <nacho _a_t_ ownyourbits _d_o_t_ com>
+# GPL licensed (see end of file) * Use at your own risk!
+#
+# More at: https://ownyourbits.com
+#
+
+install()
+{
+  apt-get update
+  apt-get install --no-install-recommends -y restic
```

Comment:
Quick note from a passerby. Perhaps also add:
```sh
restic self-update
```
Restic version available from Bullseye repository is v0.9.6 while self update will get you v0.14.0.